cmd/prometheus/main.go

```diff
@@ -68,6 +69,10 @@ var (
 		Name: "prometheus_config_last_reload_success_timestamp_seconds",
 		Help: "Timestamp of the last successful configuration reload.",
 	})
+	oldestTimestamp = prometheus.NewGauge(prometheus.GaugeOpts{
+		Name: "prometheus_oldest_timestamp_seconds",
```

Comment:
I wonder if `prometheus_tsdb_start_time_seconds` would be a better name, and whether this metric should be exported by tsdb itself.

Reply:
Yeah good point, going to follow up on https://github.com/prometheus/tsdb/pull/363.